src/data/api.js

```diff
@@ -1,9 +1,9 @@
-import api30 from './3.0/api'
-import api12 from './1.2/api'
+import api30 from './3.x/api'
+import api12 from './1.x/api'
```

Comment:
We could even go all the way and rename `api30` -> `api3x` and `api12` -> `api1x`